### ISSUE TITLE
 Latest stable version number incorrect

### ISSUE THREAD

**Issue by SamuelMarks** [cloudfoundry/nodejs-buildpack#46]
The latest stable probably now means that latest LTS, which is 4.3.0. Or non LTS is 5.6.0.

However here's what it brings down:
```
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       Resolving node version (latest stable) via semver.io...
       Downloading and installing node 0.12.9...
       Downloaded [https://pivotal-buildpacks.s3.amazonaws.com/concourse-binaries/node/node-0.12.9-linux-x64.tgz]
       Using default npm version: 2.14.9
```

**Comment by Dannyzen** [comment]
Hey @SamuelMarks thanks for the feedback. As we do not want to negatively impact users depending on 0.12.x we'll follow up with an RFC to change the default version to 4.x.

**Comment by Dannyzen** [comment]
@SamuelMarks We've created https://github.com/cloudfoundry/nodejs-buildpack/issues/50 as a follow up.

**Comment by RochesterinNYC** [comment]
Closing this as this has been implemented (https://www.pivotaltracker.com/story/show/115599647) and has been released on master and release v1.5.9 of the node-js buildpack.
